modules/ardrivo/test/Servo.cxx

```diff
@@ -0,0 +1,99 @@
+#include <catch2/catch.hpp>
+#include <range/v3/algorithm/for_each.hpp>
+#include "BoardDataDef.hxx"
+#include "Entrypoint.hxx"
+#include "Servo.h"
+
+void init_fake() {
+    auto loc_board_data = std::make_unique<BoardData>();
+    loc_board_data->servo_value = std::vector<std::atomic_uint8_t>(255);
+
+    auto loc_board_info = std::make_unique<BoardInfo>();
+    loc_board_info->pins_caps.resize(255);
+    ranges::for_each(loc_board_info->pins_caps, [](PinCapability& pin) { pin.pwm_able = true; });
+    loc_board_info->pins_caps[2].pwm_able = false;
+
+    init(loc_board_data.release(), loc_board_info.release());
```

Comment:
This leaks, just so you know; upstream will be patched at the end of the week to fix that nicely

Reply:
what do you mean?